.github/workflows/static_checks.yml

```diff
@@ -107,3 +107,8 @@ jobs:
       - name: Regex Patterns Validation Check
         run: |
```

Comment:
Should this also be ``if: always()`` in #3340?

Reply:
Done.